app/model/fee-policy/flat_fee_per_token_policy.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fee_policy
+
+type FlatFeePerTokenPolicy struct {
+	Networks  []uint64
+	TokenFees map[string]int64 // { token: "0.0.4564", value: 7 }
```

Comment:
nit: I think this comment will be better by showing the exact mapping:
`{ "0.0.4564":  7 }` otherwise it looks a bit like it's some kind of strange map which has a field token